RocketChat/Rocket.Chat
# [FIX] Custom status doesn't update properly

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce

- Add a custom status in admin menu
- Use this status in some user A and then use some user B to check the status message below the user name in a direct message

## Further comments
